collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+If you are using compute shaders, a pair of goodies should make your life way more exciting. First, is the newly added ability to create custom texture objects ([GH-79288](https://github.com/godotengine/godot/pull/79288)). You can see how it works in practice to create water effects with this [soon-to-be merged compute texture demo](https://github.com/godotengine/godot-demo-projects/pull/938). And another submission adds a set of API to call compute code on the render thread, for cases when you need to synchronize compute shaders and rendering ([GH-79696](https://github.com/godotengine/godot/pull/79696)).
+
+A prominent bug has been fixed, causing a crash if you go over 204 lights, decals, and reflection probes, combined ([GH-80845](https://github.com/godotengine/godot/pull/80845)). This issue stopped users from fully flexing the strength of the Forward+ renderer, but now they are free to use up to 512 of each entity type in the default configuration (this limit can be increased). And the built-in glow effect have been improved, making its visuals closer to the high quality look from Godot 3.x ([GH-82353](https://github.com/godotengine/godot/pull/82353) by [Raffaele Picca](https://github.com/RPicster)).
+
+And now, for the big hitters...
+
+#### New features
+
+Godot 4.2 features support for AMD's <abbr title="FidelityFX Super Resolution">FSR</abbr> 2.2, an open upscaling technique that works on GPUs of all vendors ([GH-81197](https://github.com/godotengine/godot/pull/81197)). This is possible thanks to [Darío Samo](https://github.com/DarioSamo), who also implemented a prerequisite support for motion vectors in skeletons, blend shapes, and particles ([GH-80618](https://github.com/godotengine/godot/pull/80618), [GH-80688](https://github.com/godotengine/godot/pull/80688)).
+
+Darío also implemented a new lightmapper denoising approach to replace old, extremely bulky and slow <abbr title="OpenImage Denoise">OIDN</abbr> denoiser. The new <abbr title="Joint Non-Local Means">JNLM</abbr> denoiser is much faster as it utilizes compute shaders (and thus your GPU), and it's also significantly more lightweight ([GH-81659](https://github.com/godotengine/godot/pull/81659)). While the quality difference can be noticeable under some circumstances, thanks to drastically reduced baking time you can now use higher resolution maps to counteract that.
```

Comment:
```suggestion
Darío also implemented a new lightmapper denoising approach to replace the old, extremely bulky and slow <abbr title="OpenImage Denoise">OIDN</abbr> denoiser. Unfortunately, due to build system incompatibilities, we were stuck on an old, single-threaded version of OIDN that is much slower and lower quality than the current version of OIDN. The new <abbr title="Joint Non-Local Means">JNLM</abbr> denoiser is much faster as it utilizes compute shaders (and thus your GPU), and it's also significantly more lightweight ([GH-81659](https://github.com/godotengine/godot/pull/81659)). While the quality decrease can be noticeable under some circumstances, thanks to drastically reduced baking time you can now use higher resolution maps to counteract that.
```

Reply:
I'll try to simplify this suggestion a bit, but I agree it's worth mentioning that a better version of OIDN also exists.